recipes/rdkit/build.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+PY_INC=`$PYTHON -c "from distutils import sysconfig; print (sysconfig.get_python_inc(0, '$PREFIX'))"`
+
+if [ "$OSX_ARCH" == "x86_64" ]; then
+  export CXXFLAGS="-std=c++11 -stdlib=libc++ -Wno-parentheses -Wno-logical-op-parentheses -Wno-format"
+  export CFLAGS="-Wno-parentheses -Wno-logical-op-parentheses -Wno-format"
+fi
+cmake \
+    -D RDK_INSTALL_INTREE=OFF \
+    -D RDK_INSTALL_STATIC_LIBS=OFF \
+    -D RDK_BUILD_INCHI_SUPPORT=ON \
+    -D RDK_BUILD_AVALON_SUPPORT=ON \
+    -D RDK_USE_FLEXBISON=OFF \
+    -D RDK_BUILD_CAIRO_SUPPORT=ON \
+    -D RDK_BUILD_THREADSAFE_SSS=ON \
+    -D RDK_TEST_MULTITHREADED=ON \
+    -D CMAKE_SYSTEM_PREFIX_PATH=$PREFIX \
+    -D CMAKE_INSTALL_PREFIX=$PREFIX \
```

Comment:
Kind of a minor point, but typically we quote paths just in case of spaces or other weird characters that show up. There are few lines below that fit in this category of issue as well. Just commenting on this one.